### PR TITLE
Update tutorial to cosmwasm/wasmd 0.7

### PR DESCRIPTION
**Pull request by ethanfrey**
- [x] Update cli samples for escrow demo
- [x] Update escrow code samples (rust)

### CODE REVIEW

File: docs/getting-started/edit-escrow-hints.md [hunk 1]
```diff
@@ -78,29 +84,29 @@ fn handle_steal() {
 
     // initialize the store
     let msg = init_msg(1000, 0);
-    let params = mock_params_height(&deps.api,"creator", &coin("1000", "earth"), &[], 876, 0);
-    let init_res = init(&mut deps, params, msg).unwrap();
+    let env = mock_env_height(&deps.api,"creator", &coin("1000", "earth"), &[], 876, 0);
```
Comment (webmaster128) [inline docs/getting-started/edit-escrow-hints.md:87]: Where does `mock_env_height` come from? It does not exist in the cosmwasm repo. Did we accidentally drop this feature?

File: docs/getting-started/editing-escrow-contract.md [hunk 2]
```diff
@@ -173,22 +179,26 @@ fn send_tokens<A: Api>(
...
-        log: Some(log.to_string()),
+        log: log,
         data: None,
     };
     Ok(r)
 }
 ```
 
-Note that `Params` encodes a lot of information from the blockchain, essentially providing the `Context` if you are coming from `cosmos-sdk`. This is validated data and can be trusted to compare any messages against. Refer to [the standard `cosmwasm` types](https://github.com/confio/cosmwasm/blob/master/src/types.rs#L64-L97) for references to all the available types in the environment.
+Note that `Env` encodes a lot of information from the blockchain, essentially providing the `Context` if you are coming from `cosmos-sdk`. This is validated data and can be trusted to compare any messages against. Refer to [the standard `cosmwasm` types](https://github.com/confio/cosmwasm/blob/master/src/types.rs#L62-L89) for references to all the available types in the environment.
```
Comment (webmaster128) [inline docs/getting-started/editing-escrow-contract.md:201]: When you link to lines, it would be good to replace `master` with a tag

**Comment by ethanfrey** [comment]
I'm heading off for the day and want to update the docs site asap (before more people have errors), so I will merge and publish. However, please do comment if you see any errors and I will fix them in a follow-up PR

File: docs/getting-started/editing-escrow-contract.md [hunk 3]
```diff
@@ -222,7 +238,23 @@ Both of these cases will be explained in detail in a future tutorial. But I can
...
-After we have our tested contract, we can run `cargo wasm` and produce a valid wasm output at `target/wasm32-unknown-unknown/release/escrow.wasm`. This works, but is 1.5 MB and huge for a blockchain transaction. Let's try to make it smaller.
+After we have our tested contract, we can run `cargo wasm`. But check out the size of the output:
+
+```bash
+$ cargo wasm
+$ du -h target/wasm32-unknown-unknown/release/cw_escrow.wasm
+1.8M    target/wasm32-unknown-unknown/release/cw_escrow.wasm
+```
+
+This works, but is huge for a blockchain transaction. Let's try to make it smaller. Turns out there is one flag we can add (Thanks Max):
```
Comment (webmaster128) [inline docs/getting-started/editing-escrow-contract.md:249]: "Turns out there is a linker flag to strip off debug information (Thanks Max)"

File: docs/getting-started/edit-escrow-hints.md (hunk 1 above)
Comment (ethanfrey) [reply docs/getting-started/edit-escrow-hints.md:87]: It is defined above in the escrow test cases

**Comment by ethanfrey** [comment]
Great, will update tomorrow

**Pull request merged**